src/Assertions.jl

```diff
@@ -125,11 +235,61 @@ macro v_do(s, l::Int, action)
   end
 end
 
+@doc Markdown.doc"""
+    set_verbose_level(s::Symbol, l::Int) -> Int
+
+If `s` represents a known verbosity scope, set the current verbosity level of
+`s` to `l`.
+
+One can access the current verbosity level of `s` by calling the function
+[`get_verbose_level`](@ref).
+
+If `s` is not yet known as a verbosity scope, the function raises an `ErrorException`
+showing the error message "Not a valid symbol". One can add `s` to the list of
+verbosity scopes by calling the function [`add_verbose_scope`](@ref).
+
+# Example
```

Comment:
The convention (at least in Oscar, and Julia itself) is to always call this section "Examples", so:
```suggestion
# Examples
```